request-manager/resources/request_manager_schema.sql

```diff
@@ -24,18 +24,19 @@ CREATE TABLE IF NOT EXISTS `raw_requests` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
 CREATE TABLE IF NOT EXISTS `job_log` (
-  `request_id`  BINARY(20) NOT NULL,
-  `job_id`      BINARY(4) NOT NULL,
-  `name`        VARBINARY(100) NOT NULL,
-  `try`         SMALLINT NOT NULL DEFAULT 0,
-  `type`        VARBINARY(75) NOT NULL,
-  `state`       TINYINT UNSIGNED NOT NULL DEFAULT 0,
-  `started_at`  BIGINT UNSIGNED NOT NULL DEFAULT 0, -- Unix time (nanoseconds)
-  `finished_at` BIGINT UNSIGNED NOT NULL DEFAULT 0, -- Unix time (nanoseconds)
-  `error`       TEXT NULL DEFAULT NULL,
-  `exit`        TINYINT UNSIGNED NULL DEFAULT NULL,
-  `stdout`      LONGBLOB NULL DEFAULT NULL,
-  `stderr`      LONGBLOB NULL DEFAULT NULL,
+  `request_id`   BINARY(20) NOT NULL,
+  `job_id`       BINARY(4) NOT NULL,
+  `name`         VARBINARY(100) NOT NULL,
+  `try`          SMALLINT NOT NULL DEFAULT 0,
```

Comment:
Disambiguate

Reply:
This involves renaming a column which I think the process is:

1. create a new column called job_try and change code to also set job_try
2. once deployed internally, do a backfill of try -> job_try
3. Delete `try` column

Given that the work for this takes some time, I'd rather do it in a separate PR so that it's clear where the change starts and ends.

Reply:
Sure, we can do another PR, but at this stage of development we don't really need to do a full migration and backfill. We haven't released any stable versions yet.